requirements.txt

```diff
@@ -7,6 +7,7 @@ attrs==19.1.0
 jsonpickle==1.1
 redis==3.2.1
 fakeredis==1.0.3
+dnspython==1.16.0
```

Comment:
can we also do `pymongo[tls,srv]` as mentioned here https://stackoverflow.com/a/53644085/3429596 ?

Reply:
sure, I'll update the PR.

Reply:
🥇